src/js/reducers/index.js

```diff
@@ -0,0 +1,21 @@
+import app from './app';
+import library from './library';
+import notifications from './notifications';
+import player from './player';
+import playlists from './playlists';
+import queue from './queue';
+
+const reducers = [
+    app,
+    library,
+    notifications,
+    player,
+    playlists,
+    queue,
+];
+
+export default (state, action) => {
+    return reducers.reduce((currentState, reducer) => {
+        return reducer(currentState, action);
+    }, state);
+};
```

Comment:
Any reason why you didn't use the `combineReducers` function? See http://redux.js.org/docs/api/combineReducers.html